style.css

```diff
@@ -1,3 +1,18 @@
+@media (prefers-color-scheme: dark) {
+    body {
+        background: #1e1e1e;
+        color: #cfcfcf;
+    } 
+    input, thead {
+        background: #e9e9edde !important;          
+        color: #222 !important;
+    }
+    input[disabled] {
+        background: #222 !important;
```

Comment:
This would result in foreground and background color of a disabled input field being the same, making the field entirely unreadable. Please make sure there's enough contrast either way.